File: src/main/java/com/github/kayjamlang/core/expressions/ArrayExpression.java
package com.github.kayjamlang.core.expressions;

import com.github.kayjamlang.core.opcodes.AccessType;

import java.util.List;

public class ArrayExpression extends Expression {
    /**
     * Array values
     */
    public final List<Expression> values;

    /**
     * @param values Array values
     * @param line Start line
     */
    public ArrayExpression(List<Expression> values, int line) {
        super(AccessType.NONE, line);
        this.values = values;
    }
}

File: src/main/scala/com/github/kayjamlang/core/expressions/ArrayExpression.scala
package com.github.kayjamlang.core.expressions

import com.github.kayjamlang.core.ArrayList
import com.github.kayjamlang.core.opcodes.AccessType

import scala.collection.mutable

class ArrayExpression(val values: ArrayList[Expression], line: Int) extends Expression(AccessType NONE, line)